css/timemanager.scss

```diff
@@ -642,6 +642,11 @@ label {
 			}
 		}
 	}
+
+	#app-navigation,
+	#app-content {
+		margin-top: 6px !important;
```

Comment:
I can see the `#app-content` actually doesn't need to be lowered due to how the author decided to deal with the height.

The `#app-navigation` still needs it though. But instead of add some margin, I would override the height calculation should the property is as follows:

`height: calc(100% - 56px);`

Reply:
height: calc(100% - 56px); didn't manage to lower it, I guess because it also has a position fixed it only added a gap in the bottom

Reply:
Ohh yeah, shoot, sorry, it's

```
top: 56px;
height: calc(100% - 56px);
```